apps/web/pages/settings/security/two-factor-auth.tsx

```diff
@@ -45,10 +45,12 @@ const TwoFactorAuthView = () => {
             user?.twoFactorEnabled ? setDisableModalOpen(true) : setEnableModalOpen(true)
           }
         />
-        <div>
+        <div className="!mx-4">
```

Comment:
why the !important?

Reply:
There was a conflict with a previous CSS: 
![image](https://user-images.githubusercontent.com/18273833/232586124-b29bc4d2-5e22-4c9c-8d4e-6fae4f1f23be.png)

Could not think about other solutions so went for this: 
![image](https://user-images.githubusercontent.com/18273833/232586204-d66fd9c8-a0e1-4140-9e63-3e7d687b83eb.png)

Reply:
ahh